pkg/providers/cloudstack/config/template-cp.yaml

```diff
@@ -435,6 +438,9 @@ spec:
         {{$key}}: {{$value}}
 {{- end}}
 {{- end}}
+{{- if .cloudstackEtcdAffinity }}
+      affinity: '{{ .cloudstackEtcdAffinity }}'
```

Comment:
I'm not sure if we need the quotes around the value here

Reply:
'no' value without quotes is treated as a boolean value so they are needed.